### ISSUE TITLE
csquotes loaded forcibly in English by the documentclass, clashing with otherwise Hungarian document

### ISSUE THREAD

**Issue by whisperity** [mcserep/elteikthesis#2]
When using an otherwise Hungarian document, the `csquote` package still generates "English" quotations instead of „Hungarian” ones.

MWE:

```tex
\documentclass{elteikthesis}[2020/05/15]

\usepackage{csquotes}

\title{Blah}
\date{2020}
\author{Blah}
\degree{blah}
\supervisor{Blah}
\affiliation{blah}
\university{Blah}
\faculty{Blah}
\department{Blah}
\city{Blah}
\logo{example-image-a}

\begin{document}
\documentlang{magyar}
\maketitle

\enquote{próba idézet}

\end{document}
```

![image](https://user-images.githubusercontent.com/1969470/99951349-457ed000-2d7e-11eb-8adb-edf160070e8d.png)

However, if I try to manually load the `hungarian` style in the package argument, it breaks the build because of an option clash.

```tex
\documentclass{elteikthesis}[2020/05/15]

\usepackage[hungarian]{csquotes}
```

~~~
! LaTeX Error: Option clash for package csquotes.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.10 
     
The package csquotes has already been loaded with options:
  [style=english]
There has now been an attempt to load it with options
  [hungarian]
Adding the global options:
  style=english,hungarian
to your \documentclass declaration may fix this.
Try typing  <return>  to proceed.
~~~

The suggestion `style=english,hungarian` does not work, in fact any `X` option you write there it will just keep suggesting to append the original `style=english` too.

The issue seems to be coming from here, as the `csquote` package is loaded without respect to the document's language:

https://github.com/mcserep/elteikthesis/blob/3131756e04b9f566a6ddd1da8f310ed1fa5c2f0c/elteikthesis.cls#L366

**Comment by mcserep** [comment]
Hungarian language support was only [added](https://github.com/josephwright/csquotes/commit/23b8f2a27946eac9dafd8eba3283dfb4ec88602c) in 2019 May to the `csquotes` package, that's why it is not supported in this template.

I could not get it to work in Hungarian with the `autostyle=true` option now, but I will get back to it later. Feel free to submit a PR if you have a solution meanwhile.

**Comment by mcserep** [comment]
Fixed in d74191c.
